src/cronjobs/monthlyActivity.tsx

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { SubscriberRow } from "knex/types/tables";
+import {
+  getSubscribersWaitingForMonthlyEmail,
+  updateMonthlyEmailTimestamp,
+} from "../db/tables/subscribers";
+import { initEmail, sendEmail } from "../utils/email";
+import { renderEmail } from "../emails/renderEmail";
+import { MonthlyActivityEmail } from "../emails/templates/monthlyActivity/MonthlyActivityEmail";
+import { getEmailL10n } from "../emails/getEmailL10n.node";
+import { sanitizeSubscriberRow } from "../app/functions/server/sanitize";
+import { getDashboardSummary } from "../app/functions/server/dashboard";
+import { getLatestOnerepScanResults } from "../db/tables/onerep_scans";
+import { getSubscriberBreaches } from "../app/functions/server/getSubscriberBreaches";
+import { getLocale } from "../app/functions/universal/getLocale";
+
+void run();
+
+async function run() {
+  const batchSizeEnvVar = Number.parseInt(
+    process.env.MONTHLY_ACTIVITY_EMAIL_BATCH_SIZE ?? "",
+    10,
+  );
+  const batchSize = Number.isNaN(batchSizeEnvVar) ? 10 : batchSizeEnvVar;
+  const subscribersToEmail = await getSubscribersWaitingForMonthlyEmail({
+    limit: batchSize,
+    plusOnly: true,
+  });
+  await initEmail();
+
+  await Promise.allSettled(
+    subscribersToEmail.map((subscriber) => {
+      return sendMonthlyActivityEmail(subscriber);
+    }),
+  );
+  console.log(
+    `[${new Date(Date.now()).toISOString()}] Sent [${subscribersToEmail.length}] monthly activity emails.`,
+  );
+}
+
+async function sendMonthlyActivityEmail(subscriber: SubscriberRow) {
+  // Update the last-sent date *first*, so that if something goes wrong, we
+  // don't keep resending the email a brazillion times.
+  await updateMonthlyEmailTimestamp(subscriber.primary_email);
+  const sanitizedSubscriber = sanitizeSubscriberRow(subscriber);
+  const l10n = getEmailL10n(sanitizedSubscriber);
+  const locale = getLocale(l10n);
+  /**
+   * Without an active user session, we don't know the user's country. This is
+   * our best guess based on their locale. At the time of writing, it's only
+   * used to determine whether to count SSN breaches (which we don't have
+   * recommendations for outside the US).
+   */
+  const countryCodeGuess = locale.split("-")[1] ?? "us";
+
+  const latestScan = await getLatestOnerepScanResults(
+    subscriber.onerep_profile_id,
+  );
+  const subscriberBreaches = await getSubscriberBreaches({
+    fxaUid: subscriber.fxa_uid,
+    countryCode: countryCodeGuess,
+  });
+  const data = getDashboardSummary(latestScan.results, subscriberBreaches);
```

Comment:
I still have to check with Tony whether we want to show totals, or only specifics for the past month.